presto-spi/src/test/java/io/prestosql/spi/type/TestSqlTimestamp.java

```diff
@@ -40,78 +40,6 @@ public void testBaseline()
         assertThat(newInstance(12, 0, 0).toString()).isEqualTo("1970-01-01 00:00:00.000000000000");
     }
 
-    @Test
-    public void testPositiveEpoch()
-    {
-        // round down
-        // represents a timestamp of 1970-01-01 00:00:00.111111111111
-        assertThat(newInstance(0, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00");
-        assertThat(newInstance(1, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.1");
-        assertThat(newInstance(2, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.11");
-        assertThat(newInstance(3, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.111");
-        assertThat(newInstance(4, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.1111");
-        assertThat(newInstance(5, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.11111");
-        assertThat(newInstance(6, 111111, 111111).toString()).isEqualTo("1970-01-01 00:00:00.111111");
```

Comment:
This deletion is part `Make construction of SqlTimestamp and SqlTimestampWithTimeZone stricter` commit.
Would it mnake sense toi keep the test, replacing `newInstance` with `newInstanceWithRounding` here?

Reply:
you can make `newInstanceWithRounding` package private, visiblefortesting

anyway, up to you